lib/futurism/helpers.rb

```diff
@@ -68,10 +68,18 @@ def initialize(extends:, placeholder:, options:)
         @eager = options.delete(:eager)
         @broadcast_each = options.delete(:broadcast_each)
         @controller = options.delete(:controller)
+        @updates_for_object = options.delete(:updates_for)
         @html_options = options.delete(:html_options) || {}
         @data_attributes = html_options.fetch(:data, {}).except(:sgid, :signed_params)
         @model = options.delete(:model)
+        @wrapped_for_updates_for = options.delete(:wrapped_for_updates_for)
+        if @wrapped_for_updates_for
+          @html_options[:keep] = 'keep'
```

Comment:
Needed to not add the IntersectionObserver (as it will re-request the partial over and over again when it is visible)